portal/src/locale-data/en.json

```diff
@@ -106,9 +106,12 @@
   "UsersList.column.last-login-at": "Last Login",
 
   "UsersList.disable-user": "Disable User",
+  "UsersList.enable-user": "Enable User",
 
-  "DisableUserDialog.title": "Disable User",
-  "DisableUserDialog.text": "Do you really want to disable user {username}",
+  "SetUserDisabledDialog.disableUser.title": "Disable User",
+  "SetUserDisabledDialog.disableUser.text": "Do you really want to disable user {username}?",
+  "SetUserDisabledDialog.enableUser.title": "Enable User",
+  "SetUserDisabledDialog.enableUser.text": "Do you really want to enable user {username}?",
```

Comment:
Similarly

> Enable the user {username} so that they can login again. Their existing sessions can resume.

And the buttons should be a secondary button "Cancel" and a primary button "Enable"